File: java training/day8/Person.java
import java.io.Serializable;

class Person implements Serializable{

	
	int uid;
	String name;
	int age;
	String phone;
	String email;
	
	public Person(int uid,String name,int age,String phone){
		this.uid = uid;
		this.name = name;
		this.age = age;
		this.phone = phone;

	}

}

File: day22/hello/app/src/person.ts
export class Person{
	
	name:string = "";
	age:number = 0;
	phone:string = "";
	avatar:string = "";


	constructor(name:string = 'Jagga Chaudhary',age:number = 50,phone:string = '981010000',avatar:string = 'images/adv2.jpg'){
		console.log("Person initialized!!!!!!");
		this.name = name;
		this.age = age;
		this.phone = phone;
		this.avatar = avatar;
	}


}